content/en/events/kcseu/faq.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Frequently Asked Questions (FAQ)"
+type: docs
+weight: 4
+---
+
+- [When is the summit taking place?](#when-is-the-summit-taking-place)
+- [Where is the summit located?](#where-is-the-summit-located)
+- [Will I be able to attend remotely?](#will-i-be-able-to-attend-remotely)
+- [Why is vaccination required to attend in-person?](#why-is-vaccination-required-to-attend-in-person)
+- [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I am not an Org member. Can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+May 16th, the Monday of [KubeCon Europe].
+
+[KubeCon Europe]: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/
+
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.feriavalencia.com/en/" rel="noopener noreferrer" target="_blank">Feria Valencia</a>,
+See the [Location & Venue] section for more information.
+
+[Location & Venue]: /events/past-events/2021/kcsna/location/
+
+
+### Will I be able to attend remotely?
```

Comment:
Is this information confirmed with CNCF Staff?

Reply:
It is not.  Replace with "We will not be able to support remote access"

Reply:
Better wording: "Based on our experience in Kubecon NA 2021, we do not expect to be able to support remote access for this event."